phases/19-capstone-projects/83-prompt-injection-detector/code/main.py

```
@@ -0,0 +1,247 @@
+
+
+def _try_decode_base64(text: str) -> str:
+    out = text
+    for match in re.finditer(r"[A-Za-z0-9+/=]{16,}", text):
+        tok = match.group(0)
+        try:
+            decoded = base64.b64decode(tok, validate=True).decode("utf-8", errors="ignore").lower()
+            if decoded and any(c.isalpha() for c in decoded):
+                out = out.replace(tok, " " + decoded + " ")
+        except Exception:
+            pass
```

Ruff (0.15.14)

[error] `try`-`except`-`pass` detected, consider logging the exception

(S110)